dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.2.6/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,280 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
+org.eclipse.jdt.core.compiler.compliance=1.8
```

Comment:
This needs to match the java version that is used in the current JAX-WS bundles. Should be 1.7